### PR TITLE
Chunking notebook example

### PR DESCRIPTION
**Pull request by joemcelroy**
This interactive notebook will:
- load the model "sentence-transformers__all-minilm-l6-v2" from Hugging Face and into Elasticsearch ML Node
- create an index and ingest pipeline that will chunk large fields into smaller passages and vectorize them using the model
- perform a search and return docs with the most relevant passages

Related to https://www.elastic.co/search-labs/chunking-via-ingest-pipelines

### CODE REVIEW

File: notebooks/document-chunking/with-index-pipelines.ipynb [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+    "print(client.info())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4e272c75",
+   "metadata": {},
+   "source": [
+    "## Load Model from hugging face\n",
+    "The first thing you will need is a model to create the text embeddings out of the chunks, you can use whatever you would like, but this example will run end to end on the all-distilroberta-v1 model. With an Elastic Cloud cluster created or another Elasticsearch cluster ready, we can upload the text embedding model using the eland library."
```
Comment (saarikabhasi) [inline notebooks/document-chunking/with-index-pipelines.ipynb:163]: Question: In the example below we use `sentence-transformers__all-minilm-l6-v2` but the description mentions using `all-distilroberta-v1` model.  Are both of these models same? 

Is see same in the article - https://www.elastic.co/search-labs/blog/articles/chunking-via-ingest-pipelines

Comment (joemcelroy) [reply notebooks/document-chunking/with-index-pipelines.ipynb]: thanks for spot! Yeh it uses the minilm-l6-v2 text embedding model. Yep this notebook is the article eq. its for those who want to apply this pipeline from within a notebook.

File: notebooks/document-chunking/with-index-pipelines.ipynb [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+    "      \"text\": doc[\"content\"],\n",
+    "      \"name\": doc[\"name\"]\n",
+    "    } for i, doc in enumerate(docs)\n",
+    "]\n",
+    "\n",
+    "# Add the documents to the index directly\n",
+    "response = helpers.bulk(\n",
+    "  client,\n",
+    "  operations,\n",
+    ")"
```
Comment (saarikabhasi) [inline notebooks/document-chunking/with-index-pipelines.ipynb:391]: I think it needs `time.sleep` after helpers.bulk operation to pass the`nbtest`.

**Review by saarikabhasi — APPROVED** [review]
other than some questions around clearing `nbtest`, everything else LGTM

File: notebooks/document-chunking/with-index-pipelines.ipynb (hunk 2 above)
Comment (joemcelroy) [reply notebooks/document-chunking/with-index-pipelines.ipynb:383]: added a refresh flag so docs are immediately available to the search query.

**Pull request merged**